llvm/utils/UpdateTestChecks/common.py

```diff
@@ -1635,8 +1636,9 @@ def generalize_check_lines(
         regexp = ginfo.get_regexp()
 
     multiple_braces_re = re.compile(r"({{+)|(}}+)")
+
     def escape_braces(match_obj):
-        return '{{' + re.escape(match_obj.group(0)) + '}}'
+        return "{{" + re.escape(match_obj.group(0)) + "}}"
```

Comment:
Unrelated changes?

Reply:
Unfortunately, `black` doesn't have a way to format only the diff (as far as I know), and the CI would fail without this change.